apps/settings/test/unit/panels/root/stk_item_test.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+/* globals MockNavigatorMozIccManager, MockDump,
+  MocksHelper, MockNavigatorSettings */
+requireApp('settings/shared/test/unit/load_body_html_helper.js');
+requireApp('settings/shared/test/unit/mocks/mock_navigator_moz_icc_manager.js');
+requireApp('settings/shared/test/unit/mocks/mock_dump.js');
+require('/shared/test/unit/mocks_helper.js');
+
+var mocksForIccMenu = ['Settings'];
+
+mocksForIccMenu.forEach(function(mockName) {
+  if (!window[mockName]) {
+    window[mockName] = null;
+  }
+});
+
+suite('STK Item >', function() {
+  var modules = [
+    'panels/root/stk_item',
+    'unit/mock_settings',
+    'shared_mocks/mock_l10n'
+  ];
+
+  var map = {
+    '*': {
+      'modules/settings': 'unit/mock_settings',
+      'shared/l10n' : 'shared_mocks/mock_l10n'
+    }
+  };
+
+  var realL10n, realDUMP, realMozIccManager,
+    subject, mocksHelper;
+
+  var dummySTKMenuEntries = {
+    'title': 'DummyOperator',
+    'items': [{
+      'identifier': 1,
+      'text': 'Dummy 1'
+    },
+    {
+      'identifier': 2,
+      'text': 'Dummy 2'
+    },
+    {
+      'identifier': 3,
+      'text': 'Dummy 3'
+    }],
+    'presentationType': 0
+  };
+
+  setup(function(done) {
+    var requireCtx = testRequire([], map, function() {});
+    requireCtx(modules, function(STKItem, MockSettings, MockL10n) {
+        loadBodyHTML('./_root.html');
+        realL10n = navigator.mozL10n;
+        navigator.mozL10n = MockL10n;
+        realMozIccManager = navigator.mozIccManager;
+        navigator.mozIccManager = MockNavigatorMozIccManager;
+        realDUMP = window.DUMP;
+        window.DUMP = MockDump.disable();
+
+        mocksHelper = new MocksHelper(mocksForIccMenu);
+        mocksHelper.suiteSetup();
+
+        subject = STKItem();
```

Comment:
You have to add the shim for the mock here : https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/test/unit/setup.js#L127